tests/compile-fail/collapsible_if.rs

```diff
@@ -0,0 +1,36 @@
+#![feature(plugin)]
+#![plugin(clippy)]
+
+#[deny(collapsible_if)]
+fn main() {
+    let x = "hello";
+    let y = "world";
+    if x == "hello" { //~ERROR
```

Comment:
Nit: At least part of the error message should be here. Not every `//~ERROR` must be specified, but some should be.